src/hooks/useWrapCallback.ts

```diff
@@ -37,14 +38,18 @@ export default function useWrapCallback(
     const sufficientBalance = inputAmount && balance && !balance.lessThan(inputAmount)
 
     if (inputCurrency === ETHER && currencyEquals(WETH[chainId], outputCurrency)) {
+      const { ether, weth } = getChainCurrencySymbols()
       return {
         wrapType: WrapType.WRAP,
         execute:
           sufficientBalance && inputAmount
             ? async () => {
                 try {
                   const txReceipt = await wethContract.deposit({ value: `0x${inputAmount.raw.toString(16)}` })
-                  addTransaction(txReceipt, { summary: `Wrap ${inputAmount.toSignificant(6)} ETH to WETH` })
+
+                  addTransaction(txReceipt, {
+                    summary: `Wrap ${inputAmount.toSignificant(6)} ${ether} to ${weth}`
```

Comment:
both weth and ether can be undefined here no?

Reply:
That's true, typewise, I took an assumption. That if there's a network selected theres a value. I didn't want to overcomplicate

I think is sth that shouldn't happen

Reply:
better to not make assumptions i think :)

Reply:
not anymore, now no undefined. ETH by default, which is fair assumption i think